### ISSUE TITLE
aspectsYEAHBear

### ISSUE THREAD

**Issue by userzimmermann** [coala/coala-bears#1573]
Based on @Techievena's great idea from https://github.com/coala/coala/issues/4039#issuecomment-290886365 , we should grow the `aspectsYEAHBear` that checks if the terms *aspect* and *aspects* are always written lowercase according to https://github.com/coala/cEPs/issues/69 , just like the `coalaBear` does it for the term *coala*: https://github.com/coala/coala-bears/blob/master/bears/general/coalaBear.py

The `aspectsYEAHBear` of course needs an aspect for the above main task and is always open for any ideas about additional tasks to perform.

@coala/aspects-developers

**Comment by userzimmermann** [comment]
The `aspectsYEAHBear` could also check if **aspectsYEAH** is always written correctly :)

@coala/aspects-developers Opinions?

**Comment by Asnelchristian** [comment]
yep it should, and we should probably implement Grammar( or Spelling)
aspect for the this

2017-04-01 16:40 GMT+01:00 Stefan Zimmermann <notifications@github.com>:

**Comment by Asnelchristian** [comment]
annotating the its(aspectsYEAHBear)  results right away

2017-04-01 16:43 GMT+01:00 Christian Asnel Ngoulla <ngoullasob@gmail.com>:

**Comment by userzimmermann** [comment]
@Asnelchristian Something like `Spelling.aspectsYEAH` :)

**Comment by userzimmermann** [comment]
@coala/aspects-developers So! Thanks to @Techievena we now have the `aspectsYEAH` aspect:

https://github.com/coala/coala/pull/4049
https://github.com/coala/coala/blob/master/coalib/bearlib/aspects/Spelling.py#L12

Time for the `aspectsYEAHBear`! @pratyushprakash

**Comment by jayvdb** [comment]
Creating a bear to do this, without being disabled by default in coala-quickstart, and removed from the docker, will cause errors in CI for other users repositories.

This should be merged into coalaBear (which is already special-cased to avoid this problem) , and made into a more generic tool, where the default is not going to destroy other peoples repositories who dont adhere to this rule.

**Comment by jayvdb** [comment]
"importance/high" - wtf.  Please have some perspective.  We have far more important problems.

**Comment by userzimmermann** [comment]
@jayvdb It should not be merged with the `coalaBear` but handled exactly the same way. I totally agree with you in that latter aspect :)

**Comment by jayvdb** [comment]
So now we will have a separate bear for each component of coala that wants to have its name written a certain way?

**Comment by jayvdb** [comment]
And why not share logic between coalaBear and this new aspectsYEAHBear ?

**Comment by userzimmermann** [comment]
@jayvdb This is what I also implicitly mean :) #1604 is not finished yet...
